modules/meeting/app/contracts/meetings/update_contract.rb

```diff
@@ -45,14 +48,39 @@ def user_allowed_to_edit
       end
     end
 
-    def not_before_scheduled_time
-      return unless model.recurring_meeting_id && model.scheduled_meeting
-      return unless model.changed.intersect?(%w[start_time start_date])
+    def valid_rescheduling_date # rubocop:disable Metrics/AbcSize
+      if model.start_time < Time.zone.now
+        errors.add :start_date, :after_today
+        return
+      end
```

Comment:
Minor suggestion: ActiveSupport mixes in [`DateAndTime::Calculations`](https://api.rubyonrails.org/classes/DateAndTime/Calculations.html), which gives us `past?`, `future?` and some other sugary methods:

```suggestion
      if model.start_time.past?
        errors.add :start_date, :after_today
        return
      end
```

Reply:
I feel like these mixins are less clear than `<` and `<=`, since this clearly communicates if I expect Time.zone.now to be included or not. Granted this isn't a big deal for the application, but it does matter a lot for frozen time in specs and edge cases there.